Makefile

```
@@ -139,6 +139,10 @@ frontend-lint: node_modules/.uptodate
 bddtests: python
 	@tox -qe bddtests
 
+# For a given requirements/*.in file find all the requirements/*.txt files that
+# it imports by grepping the *.in file for lines like '-r foo.txt' or '-c bar.txt'.
+find_prerequisites = $(addprefix requirements/,$(shell grep '^[[:blank:]]*\(-r\|-c\)[[:blank:]]*' "requirements/$*.in" | sed 's/^[[:blank:]]*\(-r\|-c\)[[:blank:]]*//'))
```

This can be reduced to just one `sed` command instead of a `grep '...' "requirements/$*.in | sed '...'. Something like this:

```suggestion
find_prerequisites = $(addprefix requirements/,$(shell sed 's/^[[:blank:]]*\(-r\|-c\)[[:blank:]]*//p' "requirements/$*.in"))
```